src/main/java/commands/Command.java

```diff
@@ -0,0 +1,20 @@
+package commands;
+
+import exception.FredException;
+
+import storage.Storage;
+
+import tasklist.TaskList;
+
+import ui.Ui;
+
+public abstract class Command {
+
+    protected boolean isExit;
+
+    public abstract void execute(TaskList tasks, Ui ui, Storage storage) throws FredException;
+
```

Comment:
Perhaps you can add a header here so you can simply inherit the javadocs for DeleteCommand, AddCommand, etc